api/v1alpha1/dataimporterspec/schema.yaml

```diff
@@ -0,0 +1,134 @@
+openapi: 3.0.2
+components:
+  schemas:
+    Spec:
+      type: object
+      description: |
+        defines the structure of the JSON payload that is passed to the container running the importer process.
+  
+        The importer is expected to read a path to this JSON as the first command-line argument. For example:
+
+        ./my-importer.sh /path/to/spec.json
```

Comment:
Should we already mention the correct path here? We are deciding where this file will be mounted so we might as well mention it here already.